### DISCUSSION TITLE
How could use pnpm link in overrides?

### DISCUSSION THREAD

**Discussion by mikekubn** [pnpm/pnpm#4996] [Q&A] [answered]
Hi everyone,

I have two repositories one use `pnpm` and second "package" use `npm`. I would want to use `npm link`, because I would need test changes in package.

I did something like:
- my-package used `npm link`
- my-app used `pnpm link my-package ` but I have problem with `overrides` which still use same version.

```
"pnpm": {
  "overrides": {
    "my-package-foo": "3.4.0",
     ....
  }
},
```

I tried use in `overrides` - `"my-package-foo": "link:../foo"` but this not work, too.

Anyone got any ideas, thank you.

**Comment by zkochan** [comment, marked as answer]
You can use `"my-package-foo": "link:../foo"` in the overrides, it should work. What was the problem?

**Reply by mikekubn** [reply]
I tried it again today and it works. So I don't know where was a problem when I tried first. Maybe if the project is really big, it is good to use `Developer: Reload Window` in VS Code.
